rebar.config

```diff
@@ -49,7 +49,9 @@
 {swagger_endpoints, [{src, "config/swagger.yaml"}, {dst, "apps/aeutils/src/endpoints.erl"}]}.
 
 {relx, [{release, { epoch, "version value comes from VERSION" },
-         [runtime_tools, lager, setup, sext, rocksdb, mnesia_rocksdb, gproc,
+         % sasl is required for the command `epoch versions` to work,
+         % it is disabled in `sys.config` though.
```

Comment:
This comment should be removed, I don't think it adds anything considering the review comment was about the PR itself, not the code.